### DISCUSSION TITLE
OpenTabletDriver

### DISCUSSION THREAD

**Discussion by spring-man** [OpenTabletDriver/OpenTabletDriver#2710] [Q&A] [answered]
How do i use a Artisul A601 with an open tablet driver?
![スクリーンショット 2023-04-22 145215](https://user-images.githubusercontent.com/131512197/233765498-9a7a96a7-0447-40e7-a3db-7ab82c444a3f.png)

**Comment by jamesbt365** [comment, marked as answer]
You go to help > export diagnostic, save the file, send it here so i can see why it isn't working.

**Reply by spring-man** [reply]
[opendriver.txt](https://github.com/OpenTabletDriver/OpenTabletDriver/files/11301112/opendriver.txt)

**Reply by jamesbt365** [reply]
This will be much easier supported if your join our [Discord](https://opentabletdriver.net/discord), if you can do that we can support this there.

**Comment by gonX** [comment]
Will be tracked in #3039
